communitybsl.properties

```diff
@@ -1,14 +1,20 @@
 category=Languages
 description=Code Analyzer for 1C (BSL)
 homepageUrl=https://1c-syntax.github.io/sonar-bsl-plugin-community/en
-archivedVersions=1.0.1,1.1.0,1.1.1,1.2.0,1.3.1,1.4.2,1.5.0,1.6.0
-publicVersions=1.6.1
+archivedVersions=1.0.1,1.1.0,1.1.1,1.2.0,1.3.1,1.4.2,1.5.0,1.6.0,1.6.1
+publicVersions=1.7.0
 
 defaults.mavenGroupId=com.github.1c-syntax
 defaults.mavenArtifactId=sonar-communitybsl-plugin
 
+1.7.0.description=Query highlighting, 6 new rules.
+1.7.0.sqVersions=[7.9,LATEST]
+1.7.0.date=2020-09-23
+1.7.0.changelogUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/tag/v1.7.0
+1.7.0.downloadUrl=https://github.com/1c-syntax/sonar-bsl-plugin-community/releases/download/v1.7.0/sonar-communitybsl-plugin-1.7.0.jar
+
 1.6.1.description=Fix "skipSupport" setting values, add "parameters" tag to rules with parameters.
-1.6.1.sqVersions=[7.9,LATEST]
+1.6.1.sqVersions=[7.9,7.9.4]
```

Comment:
This end range needs to be set to the latest version: 8.4.2. Otherwise users of E.G. 8.2 won't get a prompt in the Marketplace to upgrade to your latest version.

Reply:
Ah, sorry about that. Fixed!